.github/workflows/test-integrations.yml

```diff
@@ -616,7 +616,30 @@ jobs:
         run: chmod +x consul
       - name: Build consul:local image
         run: docker build -t ${{ env.CONSUL_LATEST_IMAGE_NAME }}:local -f ./build-support/docker/Consul-Dev.dockerfile .
-      - name: Peering commonTopo Integration Tests
+      - name: Peering Integration Tests - Shared Topology Tests
```

Comment:
What's the motivation for splitting them like this?

Reply:
No reason, just wanted to separate the jobs to run faster in parallel